internal/tester.go

```diff
@@ -79,63 +94,120 @@ func (t *Tester) ExecuteTests() error {
 }
 
 func executeSingleTestFile(t *Tester, tf string, timeout time.Duration, resources []config.Resource) error {
-	chainsawCommand := fmt.Sprintf(`"${CHAINSAW}" test --test-dir %s --test-file %s --skip-delete --parallel 1 2>&1`,
-		filepath.Clean(filepath.Join(t.options.Directory, caseDirectory)),
-		filepath.Clean(tf))
+	// Explicitly Set Controller Logger
+	// because of log.SetLogger(...) was never called;
+	ctrl.SetLogger(zap.New(zap.UseDevMode(true)))
 
-	ctx, cancel := context.WithTimeout(context.Background(), timeout)
-	defer cancel()
+	log.Println("Loading default configuration...")
+	configuration, err := kconfig.DefaultConfiguration()
+	if err != nil {
+		return errors.Wrap(err, "failed to load Chainsaw default configuration")
+	}
+	configuration.Spec.Discovery.TestFile = tf
+	configuration.Spec.Execution.Parallel = ptr.To(1)
+	configuration.Spec.Cleanup.SkipDelete = true
```

Comment:
Do we need to set any other configuration field for preserving the old behavior?

Reply:
before we used the binary input parameters

`${CHAINSAW}" test --test-dir %s --test-file %s --skip-delete --parallel 1`

`--test-dir %s`
which maps to `tests, err := discovery.DiscoverTests(tf, nil, false, t.options.Directory)`

--test-file %s
which maps to `configuration.Spec.Discovery.TestFile = tf`

--skip-delete 
which maps to `configuration.Spec.Cleanup.SkipDelete = true`

--parallel 1
which maps to `configuration.Spec.Execution.Parallel = ptr.To(1)`

chainsaw is using default when you not explicit set some of the inputs `configuration, err := kconfig.DefaultConfiguration()`